doc/deployment/amazon_web_services.md

```diff
@@ -115,7 +115,7 @@ $ AWS_KEY=[secret access key]
 Run the following command to deploy your Pachyderm cluster:
 
 ```shell
-$ pachctl deploy amazon ${BUCKET_NAME} ${AWS_ID} ${AWS_KEY} " " ${AWS_REGION} ${STORAGE_SIZE} --static-etcd-volume=${STORAGE_NAME}
+$ pachctl deploy amazon ${BUCKET_NAME} "" ${AWS_ID} ${AWS_KEY} " " ${AWS_REGION} ${STORAGE_SIZE} --static-etcd-volume=${STORAGE_NAME}
```

Comment:
Looks like there's still an empty string argument for the cloud formation ID there.